frontend/src/components/eMIB/EditActionDialog.jsx

```diff
@@ -64,39 +80,46 @@ class EditActionDialog extends Component {
 
   render() {
     const { showDialog, handleClose, actionType, editMode } = this.props;
+    const closeButton = (
+      <button onClick={handleClose} style={styles.closeButton}>
+        <i class="fas fa-times" />
+      </button>
+    );
     return (
       <div>
         <Modal show={showDialog} onHide={handleClose}>
           <div>
             <Modal.Header style={styles.modalHeader}>
               {
-                <div style={styles.title}>
+                <div style={styles.fullWidth}>
                   {actionType === ACTION_TYPE.email && (
-                    <div>
+                    <div style={styles.fullWidth}>
                       <i style={styles.icon} className="fas fa-envelope" />
                       <h3 style={styles.dialogHeaderText}>
                         {editMode === EDIT_MODE.create &&
                           LOCALIZE.emibTest.inboxPage.editActionDialog.addEmail}
                         {editMode === EDIT_MODE.update &&
                           LOCALIZE.emibTest.inboxPage.editActionDialog.editEmail}
                       </h3>
+                      {closeButton}
                     </div>
                   )}
                   {actionType === ACTION_TYPE.task && (
-                    <div>
+                    <div style={styles.fullWidth}>
                       <i style={styles.icon} className="fas fa-tasks" />
                       <h3 style={styles.dialogHeaderText}>
                         {editMode === EDIT_MODE.create &&
                           LOCALIZE.emibTest.inboxPage.editActionDialog.addTask}
                         {editMode === EDIT_MODE.update &&
                           LOCALIZE.emibTest.inboxPage.editActionDialog.editTask}
                       </h3>
+                      {closeButton}
```

Comment:
why not just define this inline? It doesn't seem conditional or that it's needed to be exported.

Reply:
Added inline in 3b02de2